### PR TITLE
[ZH] Updating the home page.

### PR DESCRIPTION
**Pull request by JonathanSum**

### CODE REVIEW

**Review by makeecat — COMMENTED** [review]
Can you update the link of chap 3-15 as the way chap 1-2 did?
Ex.
```html
<td><a href="{{site.baseurl}}/zh/week01/01-1">历史及动力</a></td>
```

**Comment by JonathanSum** [comment]
@makeecat Which line?

File: docs/zh/index.md [hunk 1]
```diff
@@ -256,6 +256,86 @@ translator: Mingyang Zhao
...
+        <a href="https://github.com/Atcold/pytorch-Deep-Learning/blob/master/slides/10%20-%20Attention%20%26%20transformer.pdf">🖥️</a>
+        <a href="https://github.com/Atcold/pytorch-Deep-Learning/blob/master/15-transformer.ipynb">📓</a>
+        <a href="https://youtu.be/f01J0Dri-6k">🎥</a>
+      </td>
+    </tr>
+<!-- =============================== WEEK 13 =============================== -->
+    <tr>
+      <td rowspan="3" align="center"><a href="en/week13/13">⑬</a></td>
+      <td rowspan="2">讲座</td>
+      <td><a href="en/week13/13-1">图形卷积网络 I</a></td>
```
Comment (makeecat) [inline docs/zh/index.md:283]: `<td><a href="zh/week13/13-1">图形卷积网络 I</a></td>`

File: docs/zh/index.md [hunk 2]
```diff
@@ -256,6 +256,86 @@ translator: Mingyang Zhao
...
+      <td rowspan="2">
+        <a href="https://drive.google.com/file/d/1oq-nZE2bEiQjqBlmk5_N_rFC8LQY0jQr/">🖥️</a>
+        <a href="https://youtu.be/Iiv9R6BjxHM">🎥</a>
+      </td>
+    </tr>
+    <tr><td><a href="en/week13/13-2">图形卷积网络 II</a></td></tr>
```
Comment (makeecat) [inline docs/zh/index.md:289]: `    <tr><td><a href="zh/week13/13-2">图形卷积网络 II</a></td></tr>`

File: docs/zh/index.md [hunk 3]
```diff
@@ -256,6 +256,86 @@ translator: Mingyang Zhao
         <a href="https://youtu.be/VcrCr-KNBHc">🎥</a>
       </td>
     </tr>
+    <!-- =============================== WEEK 12 =============================== -->
+    <tr>
+      <td rowspan="3" align="center"><a href="en/week12/12">⑫</a></td>
+      <td rowspan="2">Lecture</td>
+      <td><a href="en/week12/12-1">DL for NLP I</a></td>
```
Comment (makeecat) [inline docs/zh/index.md:263]: Need translation, and change href="en to href="zh

File: docs/zh/index.md [hunk 4]
```diff
@@ -256,6 +256,86 @@ translator: Mingyang Zhao
...
+      <td rowspan="2">
+        <a href="https://drive.google.com/file/d/149m3wRavTp4DQZ6RJTej8KP8gv4jnkPW/">🖥️</a>
+        <a href="https://youtu.be/6D4EWKJgNn0">🎥</a>
+      </td>
+    </tr>
+    <tr><td><a href="en/week12/12-2">DL for NLP II</a></td></tr>
```
Comment (makeecat) [inline docs/zh/index.md:269]: en to zh

File: docs/zh/index.md [hunk 5]
```diff
@@ -256,6 +256,86 @@ translator: Mingyang Zhao
...
+    <tr>
+      <td rowspan="1">Practicum</td>
+      <td><a href="en/week12/12-3">Attention & transformer</a></td>
```
Comment (makeecat) [inline docs/zh/index.md:272]: Translation needed, and en to zh

File: docs/zh/index.md (hunk 1 above)
Comment (makeecat) [inline docs/zh/index.md:283]: en to zh

File: docs/zh/index.md [hunk 6]
```diff
@@ -256,6 +256,86 @@ translator: Mingyang Zhao
...
+      <td><a href="en/week13/13-1">图形卷积网络 I</a></td>
+      <td rowspan="2">
+        <a href="https://drive.google.com/file/d/1oq-nZE2bEiQjqBlmk5_N_rFC8LQY0jQr/">🖥️</a>
+        <a href="https://youtu.be/Iiv9R6BjxHM">🎥</a>
+      </td>
+    </tr>
+    <tr><td><a href="en/week13/13-2">图形卷积网络 II</a></td></tr>
+    <tr>
+      <td rowspan="1">动手做</td>
+      <td><a href="en/week13/13-3">图形卷积网络 III</a></td>
```
Comment (makeecat) [inline docs/zh/index.md:292]: en to zh

File: docs/zh/index.md [hunk 7]
```diff
@@ -256,6 +256,86 @@ translator: Mingyang Zhao
...
+        <a href="https://github.com/Atcold/pytorch-Deep-Learning/blob/master/slides/11%20-%20GCN.pdf">🖥️</a>
+        <a href="https://github.com/Atcold/pytorch-Deep-Learning/blob/master/16-gated_GCN.ipynb">📓</a>
+        <a href="https://youtu.be/2aKXWqkbpWg">🎥</a>
+      </td>
+    </tr>
+<!-- =============================== WEEK 14 =============================== -->
+    <tr>
+      <td rowspan="3" align="center"><a href="en/week14/14">⑭</a></td>
+      <td rowspan="2">讲座</td>
+      <td><a href="en/week14/14-1">用于结构化预测的深度学习</a></td>
```
Comment (makeecat) [inline docs/zh/index.md:303]: en to zh

File: docs/zh/index.md [hunk 8]
```diff
@@ -256,6 +256,86 @@ translator: Mingyang Zhao
...
+      <td rowspan="2">
+        <a href="https://drive.google.com/file/d/1qBu-2hYWaGYEXeX7kAU8O4S2RZ1hMjsk/">🖥️</a>
+        <a href="https://youtu.be/gYayCG6YyO8">🎥</a>
+      </td>
+    </tr>
+    <tr><td><a href="en/week14/14-2">图形能量基礎方式</a></td></tr>
```
Comment (makeecat) [inline docs/zh/index.md:309]: en to zh, and consider be consistent with PR#758

File: docs/zh/index.md [hunk 9]
```diff
@@ -256,6 +256,86 @@ translator: Mingyang Zhao
...
+    <tr>
+      <td rowspan="1">动手做</td>
+      <td><a href="en/week14/14-3">正则化和贝叶斯</a></td>
```
Comment (makeecat) [inline docs/zh/index.md:312]: en to zh

File: docs/zh/index.md [hunk 10]
```diff
@@ -256,6 +256,86 @@ translator: Mingyang Zhao
...
+        <a href="https://github.com/Atcold/pytorch-Deep-Learning/blob/master/slides/08%20-%20Bayesian%20NN.pdf">🖥️</a>
+        <a href="https://github.com/Atcold/pytorch-Deep-Learning/blob/master/13-bayesian_nn.ipynb">📓</a>
+        <a href="https://youtu.be/DL7iew823c0">🎥</a>
+      </td>
+    </tr>
+<!-- =============================== WEEK 15 =============================== -->
+    <tr>
+      <td rowspan="2" align="center"><a href="en/week15/15">⑮</a></td>
+      <td rowspan="2">动手做</td>
+      <td><a href="en/week15/15-1">潜在变量基于能量的模型（EBM）的推论</a></td>
```
Comment (makeecat) [inline docs/zh/index.md:325]: en to zh, consider be consistent with PR#758

File: docs/zh/index.md [hunk 11]
```diff
@@ -256,6 +256,86 @@ translator: Mingyang Zhao
...
+      <td rowspan="1">
+        <a href="https://github.com/Atcold/pytorch-Deep-Learning/blob/master/slides/12%20-%20EBM.pdf">🖥️</a>
+        <a href="https://youtu.be/sbhr2wjU1-I">🎥</a>
+      </td>
+    </tr>
+    <tr>
+      <td><a href="en/week15/15-2">训练潜在变量能量基础模型（EBM）</a></td>
```
Comment (makeecat) [inline docs/zh/index.md:332]: en to zh

**Review by makeecat — COMMENTED** [review]
Can you also update these links by changing en to zh in previous chapters?

**Review by makeecat — COMMENTED** [review]
Please change en to zh in the link to week 5-11 as well, thanks!

**Comment by JonathanSum** [comment]
I have changed all of them to be zh/.

**Review by makeecat — APPROVED** [review]
I think we are all good.

**Review by Atcold — APPROVED** [review]
Haha, cool, you two have e-met! 😊

**Pull request merged**